WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewController.swift

```diff
@@ -129,7 +130,20 @@ private extension CustomerSelectorViewController {
                                              loadResultsWhenSearchTermIsEmpty: loadResultsWhenSearchTermIsEmpty,
                                              showSearchFilters: showSearchFilters,
                                              onAddCustomerDetailsManually: onAddCustomerDetailsManually,
-                                             onDidSelectSearchResult: onCustomerTapped),
+                                             onDidSelectSearchResult: onCustomerTapped,
+                                             onDidStartSyncingAllCustomersFirstPage: {
+                                                 Task { @MainActor [weak self] in
+                                                     guard let searchTableView = self?.searchViewController?.tableView else {
+                                                         return
+                                                     }
+                                                     self?.displayGhostContent(over: searchTableView)
+                                                 }
+                                             },
+                                             onDidFinishSyncingAllCustomersFirstPage: {
+                                                 Task { @MainActor [weak self] in
+                                                     self?.removeGhostContent()
+                                                 }
+                                             }),
```

Comment:
Non-blocker: Should we extract these two blocks to their own methods? The `addSearchViewController` method now is also handling how the ghostable content works. For example something like:

```
onDidStartSyncingAllCustomersFirstPage: { self.displayGhostContent() },
```
Where:
```
    @MainActor
    func displayGhostContent() {
        guard let searchTableView = self.searchViewController?.tableView else {
            return
        }
        self.displayGhostContent(over: searchTableView)
    }
```

Reply:
Thanks for the review @iamgabrielma ! In this case, I don't see much benefit in adding a function for these pieces of code, as they happen only there, and showing/hiding the ghost content is the only purpose of those.  Other than that, we already have a `displayGhostContent` function, declared in the `GhostableViewController` protocol, which this class conforms to. There's a difference though, when opening the screen we show the ghost-loading animation over the whole screen, in this new case we don't cover the search bar.

Reply:
Sounds good! Thanks for clarifying :D